source/index.rst

```diff
@@ -13,5 +13,4 @@ Welcome to DHIL Documentation Guide's documentation!
    intro
    tools
    markup
-   contributors
-   todo
```

Comment:
The To Do file is important, and should not be removed.

Reply:
restored.